### PR TITLE
Renovation: Fix onChange call for RadioButton inside RadioGroup

### PR DESCRIPTION
**Pull request by const314**

### CODE REVIEW

File: packages/jquery/patch/radio-button-hocs.jsx [hunk 1]
```diff
@@ -26,6 +26,7 @@ function withRadioGroup(RadioButton) {
         const handleChange = (event) => {
             event.preventDefault();
             event.stopPropagation();
```
Comment (williamvinogradov) [inline packages/jquery/patch/radio-button-hocs.jsx]: Let's not cancel the native event here:
```suggestion
```

File: packages/react/src/components/radio-button/radio-button-hocs.tsx [hunk 2]
```diff
@@ -46,6 +46,7 @@ function withRadioGroup(RadioButton: RadioButtonRenderType) {
     const handleChange: ChangeEventHandler<HTMLInputElement> = (event) => {
       event.preventDefault();
       event.stopPropagation();
```
Comment (williamvinogradov) [inline packages/react/src/components/radio-button/radio-button-hocs.tsx]: Let's not cancel the native event here:
```suggestion
```

File: packages/jquery/patch/radio-button-hocs.jsx (hunk 1 above)
Comment (const314) [reply packages/jquery/patch/radio-button-hocs.jsx]: fixed

File: packages/react/src/components/radio-button/radio-button-hocs.tsx (hunk 2 above)
Comment (const314) [reply packages/react/src/components/radio-button/radio-button-hocs.tsx]: fixed

**Review by williamvinogradov — APPROVED** [review]

**Review by IlyaKhD — APPROVED** [review]

**Pull request merged**